src/ophys_etl/transforms/decrosstalk_wrapper.py

```diff
@@ -102,6 +102,12 @@ def run(self):
                         out_file.create_dataset('roi_names', data=roi_names)
                         out_file.create_dataset('data', data=data)
 
+            for roi in plane_0.roi_list:
+                if roi.roi_id not in roi_names:
+                    msg = "ROI %d " % roi.roi_id
+                    msg += "not in final output"
+                    raise RuntimeError(msg)
```

Comment:
is this just a check at this point?
How would we reach this `RuntimeError` and, if we did, should we worry about it, and if we should, can you add some text to the user about why it might have come about?

Reply:
This is just a check. Reaching this point would be a sign that something is not behaving properly earlier in the code. I've added text to that effect in the `RuntimeError`. I can also remove this if it is too paranoid.